eve/workers/openstack-multiple-nodes/terraform/network.tf

```diff
@@ -0,0 +1,85 @@
+# Default CI network
+variable "openstack_network" {
+  type = "map"
+  default = {
+    name = "tenantnetwork1"
+  }
+}
+
+resource "openstack_networking_secgroup_v2" "nodes" {
+  name        = "${local.prefix}-nodes"
+  description = "security group for reaching out metalk8s nodes"
+}
+
+resource "openstack_networking_secgroup_rule_v2" "nodes_ssh" {
+  direction         = "ingress"
+  ethertype         = "IPv4"
+  protocol          = "tcp"
+  port_range_min    = 22
+  port_range_max    = 22
+  remote_ip_prefix  = "0.0.0.0/0"
+  security_group_id = "${openstack_networking_secgroup_v2.nodes.id}"
+}
+
+resource "openstack_networking_secgroup_rule_v2" "nodes_icmp" {
+  direction         = "ingress"
+  ethertype         = "IPv4"
+  protocol          = "icmp"
+  remote_ip_prefix  = "0.0.0.0/0"
+  security_group_id = "${openstack_networking_secgroup_v2.nodes.id}"
+
+# Main network for nodes
+variable "internal_network_range" {
+  type = "string"
+  default = "192.168.42.0/24"
```

Comment:
Once #769 lands, you'll need to create an appropriate initial configuration file on the bootstrap node.

Reply:
Yep.